generated/nidaqmx/nidaqmx.proto

```diff
@@ -4306,7 +4315,7 @@ message SetScaleAttributeDoubleArrayResponse {
 
 message SetScaleAttributeInt32Request {
   string scale_name = 1;
-  ScaleAttributes attribute = 2;
```

Comment:
I have a slight preference to change the `AttributeValues` enum name to match what you have now so the name structure for message, `Attribute` enum, and `AttributeValues` enum matches. That is `ScaleInt32AttributeValues` becomes `ScaleAttributeInt32Values`.

But if that's extra work I agree with switching the `Attributes` enum back to `ScaleInt32Attributes` for the enum names to match.

Reply:
Well,  I'd have to deal with existing code. Either by (a) having a config setting to enable the new behavior or (b) breaking compatibility (and updating existing tests/examples). So, it would add some work. Good point though.

Reply:
Thanks for the input. For this PR I changed it to match AttributeValues. I will see if I can code up a PR changing both to match the accessor methods.